### PR TITLE
Annotate std/range/package.d to please dlang/dmd#12520

### PR DESCRIPTION
**Pull request by nordlow**
Part of #8076

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -11718,7 +11718,7 @@ public:
     version (StdDdoc)
     {
         /++
-            Only defined if `isRandomAccessRange!R` is `true`.
+            Only defined if `isRandomAccesRange!R` is `true`.
```
Comment (thewilsonator) [inline std/range/package.d:11721]: this undoes a typo fix PR recently

**Review by thewilsonator — COMMENTED** [review]
otherwise looks good

**Review by RazvanN7 — APPROVED** [review]
Mind the doc typo

**Comment by nordlow** [comment]
> Mind the doc typo

Done. Ready now.

File: std/range/package.d (hunk 1 above)
Comment (nordlow) [reply std/range/package.d]: Fixed.

**Comment by thewilsonator** [comment]
removing automerge, druntime fails consistently. Maybe this needs stuff in stable?

**Comment by nordlow** [comment]
> removing automerge, druntime fails consistently. Maybe this needs stuff in stable?

Broken because of linker error triggered caused by now reverted https://github.com/dlang/phobos/pull/8103.

Rebasing to see if build passes again.

**Pull request merged**